demos/wordcount/src/main/java/com/datatorrent/demos/wordcount/ApplicationWithQuerySupport.java

```diff
@@ -38,13 +38,33 @@
 
 import org.apache.hadoop.conf.Configuration;
 
+/**
+ * Simple demo that counts word frequencies from any file dropped into a
```

Comment:
counts word frequencies? word frequencies itself is counting number of times a word occurs. I think it should find/ calculate word frequencies?